docs/docs/reference/experimental/explicit-nulls.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: doc-page
 title: "Explicit Nulls"
-movedTo: https://docs.scala-lang.org/scala3/reference/other-new-features/explicit-nulls.html
+movedTo: https://docs.scala-lang.org/scala3/reference/experimental/explicit-nulls.html
```

Comment:
this new url returns 404, the current one seems correct, I suppose a PR should be made on https://github.com/scala/docs.scala-lang first

Reply:
This is too complicated for me. I give up. Can we just remove the moveTo tag?

Reply:
I thought the idea was that we do not need to make two PRs but that it would be propagated automatically?

Reply:
> I thought the idea was that we do not need to make two PRs but that it would be propagated automatically?

@pikinier20 is working on this

Reply:
Ah, OK. Then I guess I'll close this PR for now.